### DISCUSSION TITLE
Coral accelerators in stock thread

### DISCUSSION THREAD

**Discussion by obar** [blakeblackshear/frigate#5863] [General]
There are some threads which discuss availability, but not one this really fit into. Maybe we can keep a thread going listing places that Coral can be purchased, in any form factor (one thread is [USB specific](https://github.com/blakeblackshear/frigate/discussions/546))

I had this [mini PCIe Coral](https://www.mouser.com/ProductDetail/Coral/G650-04528-01?qs=XeJtXLiO41RDsI9w8m%252Bkgg%3D%3D) on backorder from **Mouser** since May 2021 with Frigate intentions, and with occasional updates about additional delays. Today it actually shipped out. They have just under 2k to sell right now at **$26**.

One would need an adaptor to make this work for full-size PCIe; I got one from AliExpress last year for $4.25 shipped.

**Comment by mtbBrian** [comment]
I got my shipping email today as well for the same unit.  I had almost written it off after they kept pushing it out. The latest being until September. I’m not going to hold my breath until it arrives. 😉

**Reply by obar** [reply]
I've also received too many email 'updates' which amounted to them reading tea leaves to estimate an arrival date, but this is quite different: they have a package shipped out to me, arriving this week. If they also shipped yours, there's a tracking number in the email.
